appendices/devbook-guide/text.xml

```diff
@@ -40,137 +39,111 @@ tags used in a GuideXML document:
-particular example, the authors' names are enclosed in another tag <d/> a
-<c>&lt;mail&gt;</c> tag, used to specify an email address for this particular
-person. The <c>&lt;mail&gt;</c> tag is optional and can be omitted, and at
-least one <c>&lt;author&gt;</c> element is required per guide document.
+document. Next, there's a <c>&lt;guide&gt;</c> tag <d/> the entire document is
+enclosed within a <c>&lt;guide&gt; &lt;/guide&gt;</c> pair. Its <c>self</c>
+attribute must point to the relative path of the document from the root node.
+An exception is the root node itself, which has <c>&lt;guide root="true"&gt;</c>
+instead.
 </p>
 
 <p>
-Next, we come to the <c>&lt;abstract&gt;</c>, <c>&lt;version&gt;</c> and
-<c>&lt;date&gt;</c> tags, used to specify a summary of the document, the
-current version number, and the current version date (in YYYY-MM-DD format)
-respectively. Dates that are invalid or not in the YYYY-MM-DD format will
-appear verbatim in the rendered document.
+Next, there is a <c>&lt;chapter&gt;</c> tag. Every document must have exactly
+one chapter. Its <c>&lt;title&gt;</c> is used to set the title for the entire
+document.
 </p>
 
 <p>
-This sums up the tags that should appear at the beginning of a guide document.
-Besides the <c>&lt;title&gt;</c> and <c>&lt;mail&gt;</c> tags, these tags
-shouldn't appear anywhere else except immediately inside the
-<c>&lt;guide&gt;</c> tag, and for consistency it's recommended (but not
-required) that these tags appear before the content of the document.  
-</p>
-
-<p>
-Finally we have the <c>&lt;license version="3.0"/&gt;</c> tag, used to publish
-the document under the <uri link="https://creativecommons.org/licenses/by-sa/3.0/">Creative
-Commons - Attribution / Share Alike</uri> license as required by the <uri
-link="/proj/en/gdp/doc/doc-policy.xml">Documentation Policy</uri>. Historically,
-the tag <c>&lt;license /&gt;</c> was used, which denoted the 2.5 version of the
-license. This is still accepted/allowed.
+This sums up the tags that should appear at the beginning of a document.
+These tags must not appear anywhere else except immediately inside the
+<c>&lt;guide&gt;</c> tag.
 </p>
 
 </body>
 </subsection>
 <subsection>
-<title>Chapters and sections</title>
+<title>Sections and subsections</title>
 <body>
 
 <p>
 Once the initial tags have been specified, you're ready to start adding the
-structural elements of the document.  Guide documents are divided into
-chapters, and each chapter can hold one or more sections.  Every chapter and
-section has a title.  Here's an example chapter with a single section,
-consisting of a paragraph.  If you append this XML to the XML in the <uri
-link="#doc_chap2_pre1">previous excerpt</uri> and append a
-<c>&lt;/guide&gt;</c> to the end of the file, you'll have a valid (if minimal)
-guide document:
+structural elements of the document. Chapters are divided into sections, each
+section can hold one or more subsections, and so on, down to the subsubsection
```

Comment:
The expression of `and so on` is weak here since `subsubsection` is literally the next level from `subsection`, there are no intermediate levels that would have been captured by `and so on`. Also, a section can hold **zero** or more subsections, not **one**. How about:

```suggestion
section can hold zero or more subsections, which are comprised of zero or more subsubsections,
```

Reply:
Done.

Reply:
"Comprised of" isn't quite accurate, because a subsection can consist of the body alone.

I have dropped "and so on" and changed the wording to say "zero or more".